engine.py

```diff
@@ -550,12 +547,20 @@ def _emit_log_message_FUTURE(self, handler, record):
         """
 
         # Give a standard format to the message:
-        #     Shotgun DEBUG <basename>: <message>
+        #     Shotgun DEBUG (<timing>) <basename>: <message>
```

Comment:
i don't think we need the timestamps. these are now available in the log files so anyone interested can look there.